src/libstd/lib.rs

```diff
@@ -268,9 +268,13 @@ pub mod sync;
 pub mod time;
 
 #[macro_use]
+#[cfg_attr(target_os = "nacl", allow(dead_code))]
+#[cfg_attr(target_os = "nacl", allow(unused_imports))]
 #[path = "sys/common/mod.rs"] mod sys_common;
 
 #[cfg(unix)]
+#[cfg_attr(target_os = "nacl", allow(dead_code))]
+#[cfg_attr(target_os = "nacl", allow(unused_imports))]
```

Comment:
I'd prefer to not have attributes like this lying around, can this use a more targeted solution instead of sprinkling these attributes?